File: app/src/main/java/com/kotoba/takarabako/data/model/Word.kt
package com.kotoba.takarabako.data.model

data class Word(
    val id: String,
    val level: String,
    val kanji: String,
    val reading: String,
    val pos: String,
    val meaning: String,
    val exKanji: String,
    val exSegments: List<QuoteSegment>,
    val exKorean: String
)

File: KotobaNoTakarabako-iOS/Models/Word.swift
import Foundation

struct Word: Codable, Identifiable, Hashable {
    let id: String
    let level: String
    let pos: String
    let kanji: String
    let reading: String
    let meaning: String
    let exampleKanji: String
    let exampleReading: String
    let exampleKorean: String
}
